compiler/luci/pass/src/QuantizeActivation.h

```diff
@@ -27,12 +27,8 @@ namespace luci
  */
 struct QuantizeActivation final : public luci::CircleNodeMutableVisitor<void>
 {
-  QuantizeActivation(loco::DataType input, loco::DataType output)
-    : input_type(input), output_type(output)
-  {
-  }
+  QuantizeActivation(loco::DataType output) : output_type(output) {}
```

Comment:
Do we assume input dtype is always F32?

Reply:
Yes. I made the field for future usage, but it has not been used for a long time.